internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -1,13 +1,45 @@
 name: databricksreceiver
-attributes:
+resource_attributes:
   databricks.instance.name:
     description: The name of the Databricks instance as defined by the value of the "instance_name" field in the config
+    type: string
+  spark_cluster_id:
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/573c1c68f409cd6b5dc40a31f24b250a965425a3/specification/common/attribute-naming.md requests dot-delimited attributes w/ snake case for final identifying words.

Reply:
Fixed.